packages/go_router/lib/src/route.dart

```diff
@@ -655,8 +655,8 @@ class StatefulShellRoute extends ShellRouteBase {
     required this.navigatorContainerBuilder,
     this.restorationScopeId,
   })  : assert(branches.isNotEmpty),
-        assert((pageBuilder != null) ^ (builder != null),
```

Comment:
why this change?

Reply:
StatefulShellRouteData.$route will provide both pageBuilder and builder, those builder might return `null` but the builder itself is not null,